improver/spotdata/spot_manipulation.py

```diff
@@ -236,7 +236,7 @@ def process(self, cubes: CubeList) -> Cube:
                     ]
                     result = extract_subcube(result, constraint)
                 else:
-                    result = ResamplePercentiles()(
+                    result = ResamplePercentiles(skip_ecc_bounds=self.skip_ecc_bounds)(
```

Comment:
The doc-string needs a slight tweak. It currently suggests the use of `skip_ecc_bounds` only affects the conversion of probabilities to percentiles, which is where it was already passed. It is now also used in the resampling of percentiles and that should be noted.

Reply:
I've updated the doc-string here and in the related CLI.